backend/app/models.py

```diff
@@ -328,6 +330,7 @@ class Meta:
             "created",
             "updated",
             "_links",
+            "checksum",
```

Comment:
Referenced in the previous comment. 

Just a thought: not sure if the presence of `ModelLesserSchema` is justified, it is just a temporary solution - we will probably get rid of it in the future to reduce unnecessary repetitions of our Schemas.